src/Components/CommentCard.jsx

```diff
@@ -0,0 +1,14 @@
+import Comments from "./Comments";
+
+const CommentCard = (props) => {
+  const { comment } = props;
+  return (
+    <ul className="comments">
+      <li className="comment-body">{comment.body}</li>
+      <li className="comment-attributes">{comment.author}</li>
+      <li className="comment-attributes">{comment.votes} votes</li>
```

Comment:
No need for multiple li elements. You can have 1 li element containing p tags for the text you want to display